### ISSUE TITLE
IPFS-deployed NFT (BYAC) has "null"  image in our collectibles response

### ISSUE THREAD

**Issue by DmitryBespalov** [safe-global/safe-transaction-service#973]
**Describe the bug**
The image of an NFT is missing, however the image is available if you check the data using ipfs.io/ipfs browser

**To Reproduce**

Item returned from collectibles:
```
{
    "address": "redacted",
    "tokenName": "BoredApeYachtClub",
    "tokenSymbol": "BAYC",
    "logoUri": "https://safe-transaction-assets.gnosis-safe.io/tokens/logos/redacted.png",
    "id": "redacted",
    "uri": "ipfs://redacted",
    "name": null,
    "description": null,
    "imageUri": null,
    "metadata": {}
  },
```
1. Go to https://ipfs.io/ipfs/redacted
2. Then take the image id and go to https://ipfs.io/ipfs/redacted
3. You see the image.

However, the `imageUri` is null in the response above.

**Expected behavior**
Images appear for the BYAC and other ipfs-based tokens

**Environment (please complete the following information):**
 - Staging or production? production
 - Which chain? ethereum mainnet
 - OS: [e.g. iOS] web
 - Browser [e.g. chrome, safari] firefox
 - Version [e.g. 22] n/a

**Additional context**

**Comment by Uxio0** [comment]
Thanks for reporting @DmitryBespalov , it seems that something is happening with the IPFS provider we are using (cloudfare)

**Comment by Uxio0** [comment]
Closes by #976
